pkgs/tools/misc/gaphor/default.nix

```diff
@@ -79,6 +96,6 @@ buildPythonApplication rec {
     maintainers = [ ];
```

Comment:
Wanna add yourself?

Reply:
No for now, as I can't guarantee that I will keep it updated.